packages/l/llfio/xmake.lua

```diff
@@ -0,0 +1,25 @@
+package("llfio")
+    set_kind("library")
+    set_homepage("https://github.com/ned14/llfio")
+    set_description("UTF8-CPP: UTF-8 with C++ in a Portable Way")
+    set_license("Apache 2")
+
+    add_urls("https://github.com/ned14/llfio/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/ned14/llfio.git")
+    add_versions("all_tests_passed_ae7f9c5a92879285ad5100c89efc47ce1cb0031b", "a921d4c812fe3db98f0d4f7cb28c9fbdc5e2746a6b0283db5be03030557dcea6")
```

Comment:
incorrect git commit, it should be [ae7f9c5a92879285ad5100c89efc47ce1cb0031b](https://github.com/ned14/llfio/releases/tag/all_tests_passed_ae7f9c5a92879285ad5100c89efc47ce1cb0031b)

and invalid version number format, please use semver format.

```
    add_versions("2022.9.7", "ae7f9c5a92879285ad5100c89efc47ce1cb0031b")
```